version-catalog/README.md

```diff
@@ -0,0 +1,76 @@
+# SpineVersionCatalog
```

Comment:
Why do we need this README in _this_ form? It hints on something, but it's not clear when you get down to it.

What do you suggest to do about this readme? Who and when is going to do it?

Reply:
This is still a work-in-progress. I've added a template file in advance in order not to forget, while addressing other comments in the first place. I will let know when it's ready to review.